x-pack/plugins/security/server/authentication/authenticator.ts

```diff
@@ -258,6 +258,7 @@ export class Authenticator {
               ...providerCommonOptions,
               name,
               logger: options.loggers.get(type, name),
+              // @ts-expect-error
```

Comment:
Nit: I personally prefer to add some text to this comment explaining what we're trying to ignore. At a minimum, I just copy the TS error verbatim if it's not too long